plugin-ci/orb.yml

```diff
@@ -137,7 +157,7 @@ jobs:
 
   deploy-ci:
     docker:
-      - image: circleci/python:2.7
+      - image: cimg/python:2.7
```

Comment:
I don't have much context here yet, but wouldn't it be nice to bump to python 3(.10) here if the build allows it now that we're upgrading the orb's Docker containers?

Reply:
Done :+1: